docs/source/user/getting-started.rst

```diff
@@ -0,0 +1,33 @@
+Getting Started
+===============
+
+.. contents:: |toctitle|
+    :local:
+
+Installation
+------------
+
+The first step in using OpenDP is to install the OpenDP library.
+
+Installing OpenDP from PiPy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A package for OpenDP is available from `PyPI <https://pypi.org/project/opendp/>`_. You can install it using ``pip`` or other package management tool:
+
+.. code-block:: bash
+
+    % pip install opendp
+
+This will make the OpenDP modules available to your local environment.
+
+Hello, OpenDP!
+--------------
+
+Once you've installed OpenDP, you can write your first program. In the example below, we'll construct an identity ``Transformation``, then invoke it on a string.
```

Comment:
Perhaps this really is as simple as it gets but I can imagine people wondering "What is SymmetricDistance and why it it important in hello world?" They might also wonder what the "identity" transformation does in the first place. I suppose for this we could explain what it does here in the text or link to the API docs.